### PR TITLE
Feature: Add resource to Skill

### PR DESCRIPTION
**Pull request by stashdjian**
add_resource

### CODE REVIEW

File: app/actions/SkillActions.js [hunk 1]
```diff
@@ -5,16 +5,40 @@ export const SKILL = {
...
       .then(skills => dispatch({ type: SKILL.FETCH.SUCCESS, skills }))
       .catch(errors => dispatch({ type: SKILL.FETCH.FAILURE, errors }));
   };
 }
+
+export function resourceAdd(skillSlug, resource) {
+  return (dispatch) => {
+    dispatch({ type: SKILL.ADD_RESOURCE.START });
+
+    return fetch(`${config.skills_api_url}/${skillSlug}/resources.json`,
```
Comment (williamcabrera4) [inline app/actions/SkillActions.js]: Hey @stashdjian 
you can check the `fetchHelper.js` that can help to construct the options for a `POST` using fetch

Comment (karolsojko) [inline app/actions/SkillActions.js]: good idea @williamcabrera4 and nice way to encapsulate those requestes - what do you think @stashdjian ? :)

File: config.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  */
 
 module.exports = {
-  skills_api_url: 'http://skills-staging.unleash.x-team.com/api/v1/skills.json',
```
Comment (karolsojko) [inline config.js]: any reason for moving the `.json` extension from the config to action files?

Comment (stashdjian) [inline config.js]: The resources endpoint at the skills API is at api/v1/skills/resources, so storing skills.json as base path for the skills API wasn't working for me. Too specific.  So with a base path as api/v1/skills we can ask for skills finishing with just .json or go to /resources.json.

File: app/actions/SkillActions.js (hunk 1 above)
Comment (stashdjian) [inline app/actions/SkillActions.js]: Thanks @williamcabrera4, i bet we'll be using this quite often. Adopting!

**Comment by szymonmichalak** [comment]
Looks good 👍 Not sure about those comments (for docs) for each method - should we keep them and implement to all the other files?

**Comment by karolsojko** [comment]
Fixes #289

**Pull request merged**